cli/src/cmd/install.rs

```diff
@@ -42,6 +42,12 @@ pub(crate) fn install(
     dependencies: Vec<Dependency>,
     opts: DependencyInstallOpts,
 ) -> eyre::Result<()> {
+    if dependencies.is_empty() {
+        let mut cmd = Command::new("git");
+        cmd.args(&["submodule", "update", "--init", "--recursive"]);
+        cmd.spawn()?.wait()?;
+    }
```

Comment:
great

Reply:
Yep that's my understanding, right @brockelmore ?

Reply:
yes that's the thought (if there are no new deps being added)